proteus/cmeshToolsModule.cpp

```diff
@@ -1728,12 +1743,12 @@ static PyObject* SparsityInfo_getCSR(SparsityInfo *self,
 //     }
   int nnz=0;
   int dim[1];
-  dim[0] = int(self->columnIndecesMap.size())+1;
+  dim[0] = self->columnIndecesMap.size()+1;
```

Comment:
In retrospect, the commit for the memory leak fix was sloppy. I have to double check whether this change was necessary.